core/spec/models/spree/shipment_spec.rb

```diff
@@ -55,38 +55,6 @@
     expect(shipment).to be_backordered
   end
 
-  context '#determine_state' do
```

Comment:
this method is deleted now and these specs are bad anyway. at the root of it they seem to just be testing whether or not `return if true` works.